arangod/Replication2/ReplicatedLog/ReplicatedLogIterator.h

```diff
@@ -37,6 +37,9 @@
 #pragma warning(pop)
 #endif
 
+#include <velocypack/Iterator.h>
+#include <velocypack/velocypack-aliases.h>
```

Comment:
We should try to not include the `velocypack-aliases.h` in a header file. Additionally we can simply forward declare `arangodb::velocypack::ArrayIterator` in the header and get rid of the include as well.

Reply:
Solving this by moving the new class to `Utilities.h`.